addon/components/ivy-videojs-player.js

```diff
@@ -1,6 +1,41 @@
 import Ember from 'ember';
 import videojs from 'videojs';
 
+const Button = videojs.getComponent('Button');
+
+class SwitchMediaButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+  }
+  buildCSSClass() {
+    return `vjs-switch-media-button ${super.buildCSSClass()}`;
+  }
+}
+
+class SkipForwardButton extends Button {
+  constructor(player, options) {
+    super(player, options);
+  }
+  buildCSSClass() {
+    return `vjs-skip-forward icon pref10 ${super.buildCSSClass()}`;
```

Comment:
Assuming you mean previous, `pref10` should be `prev10`? Why is the number 10 hardcoded here? Why is this class needed when you already have `vjs-skip-forward`?

Looking at it now, it is even stranger... Why have class `pref10` on a skip forward button? That's the opposite direction...

Reply:
As usual everything has a reason. The reason here is that the classes "icon pref10" add a custom icon. This is how it was delivered by Inpoint. You will not find the css class here its in the frontend.

vjs-skip-forward is an videojs class.